src/resmom/mom_main.c

```diff
@@ -9151,6 +9161,9 @@ main(int argc, char *argv[])
 	 **	that is exec'ed will not have SIG_IGN set for anything.
 	 */
 	sigaction(SIGPIPE, &act, NULL);
+#ifdef PBS_UNDOLR_ENABLED
+	act.sa_handler = catch_sigusr1;
+#endif
 	sigaction(SIGUSR1, &act, NULL);
 #ifdef	SIGINFO
```

Comment:
sa_handler is now set to catch_sigusr1 for all following sigaction calls, for example for SIGINFO.  You probably do not want that.